.eslintrc

```diff
@@ -55,18 +83,49 @@
     "node": true
   },
   "parserOptions": {
+    "ecmaVersion": 2018,
```

Comment:
probably should bump this down to a lower version.

Reply:
Why bump it to a lower version? I think we want the latest version or the latest features won't work. I believe a higher version includes support for all lower versions.

Reply:
@chrisbreiding agreed